ConnectionsPanel.js

```diff
@@ -248,6 +257,7 @@ class ConnectionsPanel extends React.PureComponent {
                 withArrow={false}
                 buttonStyle={{margin: 0, padding: 0}}
                 key={cat.category}
+                isMainMenu={this.props.connectionsMode === null}
```

Comment:
I like the name of this prop. May as well define it before line 242 so you can use it to define `enText`.